File: cfgs/pythoncfgs/MAXIMUM_HEIGHT_OF_TRIANGULAR_ARRANGEMENT_OF_ARRAY_VALUES.py
Nodes
1;statement;foo_1 = 1;8;
2;conditional;foo < foo;9;
3;statement;return foo_1;15;
4;statement;foo_1 = foo_2 * (foo_3 + 1) / 2;10;
5;conditional;foo_1 < foo_2;11;
6;statement;foo_1 = foo_2;12;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;5;
4;3;6;
5;2;
6;

File: cfgs/java_cfgs_foo/MAXIMUM_HEIGHT_OF_TRIANGULAR_ARRANGEMENT_OF_ARRAY_VALUES.java
Nodes
1;special;;;
2;statement;int result = 1;14;
3;statement;int i = 1;15;
4;conditional;foo_1 <= foo_2;16;
5;statement;int y = (foo_1 * (foo_2 + 1)) / 2;18;
6;conditional;foo_1 < foo_2;19;
7;statement;result = foo_1;19;
8;statement;break;20;
9;statement;++i;17;
10;statement;return foo_1;22;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;10;
4;6;
5;7;8;
6;9;
7;10;
8;4;
9;11;
10;
